### ISSUE TITLE
react-native run-android build crashes

### ISSUE THREAD

**Issue by CptMaumau** [react-community/react-native-image-picker#662]
It it crashing because of this override in ImagePickerPackage.java

```
@Override
public List<Class<? extends JavaScriptModule>> createJSModules() {
    return Collections.emptyList();
}
```

Other projects removed it like here https://github.com/evollu/react-native-fcm/issues/470

**Comment by vbuch** [comment]
https://github.com/react-community/react-native-image-picker/pull/661

**Comment by janicduplessis** [comment]
Fixed in 0.26.4
